test/fixtures/privileges.sql

```diff
@@ -62,9 +62,12 @@ GRANT USAGE ON SEQUENCE
 TO postgrest_test_anonymous;
 
 -- Privileges for non anonymous users
+GRANT USAGE ON SCHEMA test TO postgrest_test_default_role;
 GRANT USAGE ON SCHEMA test TO postgrest_test_author;
 GRANT ALL ON TABLE authors_only TO postgrest_test_author;
 
 GRANT SELECT (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
 GRANT INSERT (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
 GRANT UPDATE (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
+
+ALTER ROLE postgrest_test_authenticator set search_path = 'public', 'test';
```

Comment:
For the tests I proved that It actually does, for example if you do a ```alter role postgrest_test_authenticator set search_path=' '``` some tests fail because they depend on the ```public``` schema being present on the ```search_path```.

I set the ```search_path``` to ```public, test``` to prove that the proc return type now doesn't depend on schema qualification. Being that ```clients``` is in the test schema.